packages/renderer/src/lib/dashboard/DashboardPage.svelte

```diff
@@ -96,8 +97,18 @@ function getInitializationContext(id: string): InitializationContext {
             <ProviderStopped provider="{providerStopped}" />
           {/each}
         {/if}
-        <LearningCenter />
-        <FeaturedExtensions />
+      </div>
+      <div class="px-2 mt-5 h-full">
+        <Card title="Learning center">
+          <svelte:fragment slot="content">
+            <LearningCenter />
+          </svelte:fragment>
+        </Card>
+        <Card title="Featured extensions">
+          <svelte:fragment slot="content">
+            <FeaturedExtensions />
+          </svelte:fragment>
+        </Card>
```

Comment:
I would be really cautious there

For example Learning center does not match the expected design
see https://github.com/containers/podman-desktop/issues/6221